src/sumologic-span-processor/index.spec.ts

```diff
@@ -481,6 +481,10 @@ describe('SumoLogicSpanProcessor', () => {
     spanProcessor.onEnd(longtaskSpan);
 
     expect(longtaskSpan.attributes['http.action_type']).toBe('document_loads');
+    expect(longtaskSpan.attributes['root_span.operation']).toBe('documentLoad');
+    expect(longtaskSpan.attributes['root_span.http.url']).toBe(
+      'https://www.unit-test-example.com/',
+    );
```

Comment:
This unit test change doesn't properly test the proposed source code changes. If you copy these changes into the `master` branch, the test will pass. It means that the proposed changes are not covered in tests.
Perhaps, writing another unit test that doesn't end root span and tests child spans for enrichment would cover this PR changes.

Reply:
Yes, this change is not relevant with fix in this PR, I just noticed that this assertion was missed.
Proper test suite for fix in this PR is still in progress :)

Reply:
@KrysKruk missing test added.